webpack.config.js

```diff
@@ -2,35 +2,32 @@ const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
-  mode: 'development',
   entry: './src/index.js',
-  output: {
-    filename: 'main.js',
-    path: path.resolve(__dirname, 'dist'),
+  mode: 'development',
+  devServer: {
+    static: './dist',
   },
-
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
   module: {
     rules: [
       {
         test: /\.css$/i,
         use: ['style-loader', 'css-loader'],
       },
-      {
-        test: /\.(png|svg|jpg|jpeg|gif)$/i,
-        type: 'asset/resource',
-      },
     ],
   },
-
-  plugins: [
-    new HtmlWebpackPlugin({
-      template: './src/index.html',
-    }),
-  ],
-
-  devtool: 'inline-source-map',
-  devServer: {
-    static: './dist',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+    publicPath: '/webpack-demo/',
   },
-
+  // type: 'module',
+  // optimization: {
+  //     runtimeChunk: 'single',
+  // },
 };
+// webpack file
```

Comment:
Suggestion:
✔️Do you have a reason for not deleting this commented code? 😁

Reply:
@Roseokpe 
Thanks for the helpful review and feedback